blosc/stune.c

```diff
@@ -161,3 +161,25 @@ void blosc_stune_update(blosc2_context * context, double ctime) {
 void blosc_stune_free(blosc2_context * context) {
     BLOSC_UNUSED_PARAM(context);
 }
+
+int split_block(blosc2_context *context, int32_t typesize, int32_t blocksize) {
+  switch (context->splitmode) {
+    case BLOSC_ALWAYS_SPLIT:
+      return 1;
+    case BLOSC_NEVER_SPLIT:
+      return 0;
+    case BLOSC_FORWARD_COMPAT_SPLIT:
+    case BLOSC_AUTO_SPLIT:
+      // These cases will be handled later
+      break;
+    default:
+      BLOSC_TRACE_WARNING("Unrecognized split mode.  Default to BLOSC_FORWARD_COMPAT_SPLIT");
+  }
+
+  int compcode = context->compcode;
+  return (
+          // Fast codecs like blosclz and lz4 always prefer to always split
+          ((compcode == BLOSC_BLOSCLZ) || (compcode == BLOSC_LZ4)) &&
+          (typesize <= MAX_STREAMS) &&
+          (blocksize / typesize) >= BLOSC_MIN_BUFFERSIZE);
+}
```

Comment:
The convention is that last line in files should always be an empty line.

Reply:
Fixed in 5d9f75c603b6fcb84ccb84bb3a00e49d42386a7f.